src/styles/MyStyles.js

```diff
@@ -12,41 +13,41 @@ const useMyStyles = makeStyles((theme) => ({
     minHeight: "40%",
     textTransform: "uppercase",
   },
-  grid__Page: {
-    width: "80%",
-  },
-  box__Page: {
-    backgroundColor: "#D8DDE1",
-    padding: "0 1rem 1rem 1rem",
-    width: "90%",
-    border: "1px solid black",
-    textAlign: "center",
-  },
-  card__TeamMember: {
-    marginBottom: ".1rem",
-  },
-  cardContent__TeamMember: {
+  // team page
```

Comment:
if file is growing bigger maybe (and it will probably) good idea would be to divide it in sections: footer, header, teamPage, XPage, YPage, commons, etc

Reply:
Ok, I split styling into separate files - this will be growing for sure.